app/breast-screening-reporting/2025/12/using-design-sprints-to-help-us-learn-quickly/index.md

```diff
@@ -0,0 +1,68 @@
+
+## Why we use design sprints
+
+The time-boxed, structured process of a design sprint brings the right people together to understand a problem, explore options, make a decision, and test it quickly.
+
+As we look at evolving the data and reporting in BSIS, our design sprints will focus on a single BSIS report as a problem space, allowing the team to run an entire discovery and early alpha in as little as one week.
+
+Each sprint brings together:
+
+- key users
+- stakeholders
+- subject matter experts (SMEs)
+- the full team – engineering, product, design, and delivery
+
+This ensures that expertise, data constraints, and delivery risks are considered in every activity.
+
+**The benefits include:**
+
+- Fewer surprises later, as technical, data, and delivery constraints are identified early
+- Investing in little time upfront leaves more time later to build, improve or even change the solution if needed
+- The whole team getting close to the problem and the users, with team members working together across all activities which creates clarity and focus
+- A change from the usual day-to-day rhythm gives the team dedicated time and space to ideate and creates momentum for the quarter
+
+## What we took away from the KC63 design sprint
+
+For our first design sprint in July 2025, we focused on understanding the simplest report in BSIS – KC63. This report compiles coverage data from all breast screening services in England. Coverage helps us understand how well the screening programme is reaching the eligible population and is formally published annually for public accountability.
+
+We explored how KC63 could be improved and delivered in FDP with the underlying question to answer being ‘does FDP’s have the capabilities we need to replace this report?’.
+
+We found that the report in BSIS was not used regularly, it lacked helpful visualisations of the data and must be used in combination with other reports to gain meaningful insights. We have found that this is a theme with a lot of the BSIS reports; that many individual tables could be replaced with fewer, more holistic insights.
+
+By the end of the design sprint we had iterated and validated a prototype dashboard that shows:
+
+- Key metrics together with uptake, coverage, and round length
+- Appropriate benchmarking and trend analysis
+- The ability to compare regions and Breast Screening Offices (BSOs)
+- Infographics, like maps, showing performance at a glance
+- Breakdowns by deprivation
+
+We are building the first, very basic version of this dashboard in FDP as Breast Screening Coverage (KC63). This helps us understand how to deliver an early version of the dashboard and the underlying setup needed for the final version.
+
+It also lays the groundwork for much more complex reports like FRQA (Film Reading Quality Assurance), which we will look at next.
```

Comment:
Feels odd to start the last para with "It" - merge into previous para, or reword?


```suggestion
Next we will look at more complex reports like Film Reading Quality Assurance (FRQA).
```